runner/Cargo.toml

```diff
@@ -11,5 +11,6 @@ default-features = false
 features = ["llvm14-0"]
 
 [dependencies]
+qir-stdlib = { path = "../stdlib", features = ["ir-range-support"] }
```

Comment:
@msoeken FYI, this change (if merged) would require you to add the feature in your Cargo.toml like it does here.